spotify_player/src/command.rs

```diff
@@ -355,6 +358,8 @@ impl Command {
             Self::MovePlaylistItemUp => "move playlist item up one position",
             Self::MovePlaylistItemDown => "move playlist item down one position",
             Self::CreatePlaylist => "create a new playlist",
+            _ => unreachable!(),
```

Comment:
```suggestion
            Self::VolumeChange { .. } => unreachable!(),
```

enum handling is preferred to be explicit to avoid misuse, e.g. forgetting to handle new variants